org.simondalvai.ball2box.yml

```diff
@@ -0,0 +1,39 @@
+app-id: org.simondalvai.ball2box
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: 3.5
+command: godot-runner
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+modules:
+  - name: Ball2Box
+    buildsystem: simple
+    build-commands:
+      # The game itself
+      - tar -xvf ball2box.tar.gz
```

Comment:
This will be handled by flatpak https://github.com/flathub/flathub/pull/4956/files#r1493086842